lib/scrabble.rb

```diff
@@ -1,7 +1,14 @@
 class Scrabble
 
   def score(word)
-    1
+    letter_values = point_values
```

Comment:
You don't need to assign this method return to a variable; since the method returns a hash, you can just access keys and values by calling the method where you need it